addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -497,7 +497,7 @@
 		}
 		"424"	//Tomislav
 		{
-			"desp"			"Tomislav: {positive}+25% extra switch speed, 25% extra spin up time, {negative}-70% health from healers"
+			"desp"			"Tomislav: {positive}+25% additional switch speed, 25% extra spin up time, {negative}-70% health from healers"
```

Comment:
Would also change `25% extra spin up time` to `additional 25% faster spin up time`.
Also `additional 25% faster weapon switch` would be better to match same with spin up?